apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -9,7 +9,11 @@ var SELECTORS = Object.freeze({
   container: '#messages-container',
   callButton: '#messages-call-number-button',
   // TODO: Remove the next selector once Message has its own accessors
-  messageContent: '.message-content > p:first-child'
+  messageContent: '.message-content > p:first-child',
+  createNewContact: 'form.contact-prompt [data-l10n-id="createNewContact"]',
```

Comment:
nit: I don't think we need ```form``` in the selector + please rename property to smth like ```createNewContactMenuItem``` or ```createNewContactMenuOption``` so that it's clear what it exactly is.